### PR TITLE
Update istioctl docs

### PR DESCRIPTION
**Pull request by ostromart**
Remove example using manifest generate to install. Remove operator production warning.

### CODE REVIEW

**Comment by ostromart** [comment]
Related https://github.com/istio/istio/pull/24509.

File: content/en/docs/setup/install/istioctl/index.md [hunk 1]
```diff
@@ -83,6 +86,12 @@ to install the `demo` profile:
 $ istioctl install --set profile=demo
 {{< /text >}}
 
+## Check what's installed
+
+`Istioctl` saves a copy of the CR that was used to install Istio into the cluster, called `installed-state`. You can
```
Comment (ericvn) [inline content/en/docs/setup/install/istioctl/index.md:91]: I believe `istioctl` should be lower case the same as with your Note above.

File: content/en/docs/setup/install/istioctl/index.md [hunk 2]
```diff
@@ -83,6 +86,12 @@ to install the `demo` profile:
...
+inspect this CR if you ever lose track of what is installed in a cluster. This CR is used to perform checks in some
+`istioctl` commands and should not be removed.
+
```
Comment (ericvn) [inline content/en/docs/setup/install/istioctl/index.md:94]: Does it make sense to have to command to output the `installed-state` CR?

File: content/en/docs/setup/install/istioctl/index.md (hunk 1 above)
Comment (ostromart) [reply content/en/docs/setup/install/istioctl/index.md:91]: Thanks, done.

File: content/en/docs/setup/install/istioctl/index.md (hunk 2 above)
Comment (ostromart) [reply content/en/docs/setup/install/istioctl/index.md:94]: Probably too trivial...and the page is already really long.

File: content/en/docs/setup/install/istioctl/index.md [hunk 3]
```diff
@@ -40,6 +40,9 @@ using the following command:
 $ istioctl install
 {{< /text >}}
 
+Note: `istioctl install` and `istioctl manifest apply` are exactly the same command - `install` was added in 1.6 as a
+simplification and `manifest apply` is deprecated and will be removed in 1.7.
```
Comment (frankbu) [inline content/en/docs/setup/install/istioctl/index.md:44]:
```suggestion
{{< tip >}}
Note that `istioctl install` and `istioctl manifest apply` are exactly the same command. In Istio 1.6, the simpler `install`
command replaces `manifest apply`, which is deprecated and will be removed in 1.7.
{{< /tip >}}
```

File: content/en/docs/setup/install/istioctl/index.md [hunk 4]
```diff
@@ -83,6 +86,12 @@ to install the `demo` profile:
 $ istioctl install --set profile=demo
 {{< /text >}}
 
+## Check what's installed
+
+`istioctl` saves a copy of the CR that was used to install Istio into the cluster, called `installed-state`. You can
+inspect this CR if you ever lose track of what is installed in a cluster. This CR is used to perform checks in some
+`istioctl` commands and should not be removed.
```
Comment (frankbu) [inline content/en/docs/setup/install/istioctl/index.md:93]:
```suggestion
The `istioctl` command saves the `IstioOperator` CR that was used to install Istio in a copy of the CR named `installed-state`.
You can inspect this CR if you lose track of what is installed in a cluster.

The `installed-state` CR is also used to perform checks in some `istioctl` commands and should therefore not be removed.
```

File: content/en/docs/setup/install/istioctl/index.md [hunk 5]
```diff
@@ -182,28 +191,21 @@ $ istioctl profile diff default demo
...
 
 {{< text bash >}}
 $ istioctl manifest generate > $HOME/generated-manifest.yaml
 {{< /text >}}
 
-Inspect the manifest as needed, then apply the manifest using this command:
+The generated manifest can be used to both inspect exactly what is installed, and track changes to the manifest
+over time. While the `IstioOperator` CR represents the full user configuration and is sufficient for tracking them,
+the output from `manifest generate` also captures any changes in the underlying charts and can be used where
+tracking of the actual installed resources is desired.
```
Comment (frankbu) [inline content/en/docs/setup/install/istioctl/index.md:204]:
```suggestion
The generated manifest can be used to inspect what exactly is installed as well as to track changes to the manifest
over time. While the `IstioOperator` CR represents the full user configuration and is sufficient for tracking it,
the output from `manifest generate` also captures possible changes in the underlying charts and therefore can be
used to track the actual installed resources.
```

File: content/en/docs/setup/install/istioctl/index.md [hunk 6]
```diff
@@ -182,28 +191,21 @@ $ istioctl profile diff default demo
...
-
-{{< warning >}}
-While `istioctl install` will automatically detect environment specific settings from your Kubernetes context, `manifest generate` cannot as it runs offline, which may lead to unexpected results. In particular, you must ensure that you follow [these steps](/docs/ops/best-practices/security/#configure-third-party-service-account-tokens) if your Kubernetes environment does not support third party service account tokens.
-{{< /warning >}}
-
-{{< tip >}}
-This command might show transient errors due to resources not being available in
-the cluster in the correct order.
-{{< /tip >}}
+The output from `manifest generate` can also be used to install Istio using `kubectl apply` or equivalent. Note that
```
Comment (frankbu) [inline content/en/docs/setup/install/istioctl/index.md:206]:
```suggestion
The output from `manifest generate` can also be used to install Istio using `kubectl apply` or equivalent. However,
```

File: content/en/docs/setup/install/istioctl/index.md [hunk 7]
```diff
@@ -182,28 +191,21 @@ $ istioctl profile diff default demo
...
+these alternative installation methods may not apply the resources with the same sequencing of dependencies as
+`istioctl install` and are not tested as part of a release.
```
Comment (frankbu) [inline content/en/docs/setup/install/istioctl/index.md:208]:
```suggestion
`istioctl install` and are not tested in an Istio release.
```

**Pull request merged**